samples/msal-node-samples/client-credentials-distributed-cache/src/AuthProvider.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import axios from 'axios';
+import { performance } from "perf_hooks";
+import { RedisClientType } from "redis";
+import {
+    Configuration,
+    LogLevel,
+    ConfidentialClientApplication,
+    ClientCredentialRequest,
+    AuthenticationResult
+} from "@azure/msal-node";
+
+import CustomCachePlugin from "./CustomCachePlugin";
+import RedisClientWrapper from "./RedisClientWrapper";
+import AxiosHelper from './AxiosHelper';
+
+export type AppConfig = {
+    instance: string;
+    tenantId: string;
+    clientId: string;
+    clientSecret: string;
```

Comment:
Please add a comment stating that production apps should use a certificate and link to the cert sample.

Reply:
added a comment and also updated the README